shared/podman/systemd.go

```diff
@@ -116,3 +116,27 @@ func EnableService(service string) error {
 	}
 	return nil
 }
+
+// Create new conf file
```

Comment:
```suggestion
// Create new systemd service configuration file.
```

Note that `godoc` is picky about sentences: they need to end with a dot.